public/index.html

```diff
@@ -1,21 +1,19 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
+
+<head>
```

Comment:
why are all the indents changed?

Reply:
b/c I have autoformat on save = true

Reply:
what settings do you want to use for html formatting?

Reply:
added inner indent setting

```
    "html.format.indentInnerHtml": true
```